.github/workflows/test-e2e-service-stream.yml

```diff
@@ -5,17 +5,8 @@ on:
 
 env:
   GO_VERSION: 1.22.2
-  TEST_PARALLELISM: 4
+  TEST_PARALLELISM: 8
```

Comment:
`16`? Or we can wait and increase it more, when test caching will be ready.

Reply:
We can increase it so at least we see how much it is running faster.